src/hearings/models/hearingActualsMainModel.ts

```diff
@@ -27,14 +27,13 @@ export interface OrganisationDetailsModel {
 }
 
 export interface PlannedDayPartyModel {
-  partyId: string;
+  partyID: string;
   partyRole: string;
   individualDetails?: IndividualDetailsModel;
   organisationDetails?: OrganisationDetailsModel;
   partyChannelSubType: string;
 }
 
-
 export interface ActualDayPartyModel {
   actualPartyId: string;
```

Comment:
it's more of just checking that we have just changed from partyId to PartyID for planningDayParty model but not for ActualDayParty modal, so will this be handled later on ?

Reply:
this is to map with real API, in back end they use actualPartyId, but another one they use PartyID, which is insistent but we have to map with them.